gradlew

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
```

Comment:
Hmn... Just realized this is back to being unaddressed... The funny thing about automated scripts is that they tend to be overwritten automatically...  I'll have to figure out the hook to modify it.